### DISCUSSION TITLE
Why do we make constant a function in the test file?

### DISCUSSION THREAD

**Discussion by DappCoderr** [Cyfrin/foundry-full-course-cu#1813] [General]
MINIMUM_USD is constant in the contract, so why do we make it a function in the test file?
`assertEq(fundMe.MINIMUM_USD(), 5e18);`

**Comment by heheboii11** [comment]
It is better to check every function in the contract, just for safety

**Comment by EngrPips** [comment]
Hello @DappCoderr, every public variable in a contract gets a getter function in solidity.
